examples/atari/src/atari_wrappers.py

```diff
@@ -304,6 +307,124 @@ def __getitem__(self, i):
         return self._force()[..., i]
 
 
+class RawTrajectoryWrapper(Wrapper):
```

Comment:
can you please add this wrapper to `catalyst.rl.wrappers`?
you need to create new module dir with `__init__.py` and `raw_trajectory.py`

I think this wrapper would be used very much.